packages/serializer/src/index.js

```diff
@@ -326,7 +328,8 @@ const jsxMark = {
 }
 
 const jsxBlockTypes = {
-  youtube: '<YouTube />'
+  youtube: '<YouTube />',
+  TomatoBox: '<TomatoBox />'
```

Comment:
This will be the type I use to test this in a subsequent PR.

Reply:
So if someone *does* use a `TomatoBox` component, will this break?

Reply:
It would once we officially support custom components. So we'll prolly have to namespace any "builtins" in some way.